include/inviwo/core/util/clock.h

```diff
@@ -187,7 +187,7 @@ void ScopedClock<Clock>::printAndReset() {
 }
 
 /**
- * \class ScopedClockCPU
+ * \class inviwo::ScopedClockCPU
```

Comment:
hm, maybe this should be mentioned in the developer docs that the namespace is required here @xeTaiz